simtools/corsika/corsika_default_config.py

```diff
@@ -0,0 +1,217 @@
+        ):
+            raise ValueError(
+                f"The zenith angle, {zenith_angle:.1f}, "
+                f"is outside of the allowed interval, {allowed_zenith_angle_interval}."
+                "This interval is enforced because values can only be interpolated "
+                "between zenith angles that were manually optimized."
+            )
+
+        self._zenith_angle = zenith_angle
+
+    @property
+    def energy_slope(self):
+        """
+        Energy slope.
+        """
+        return self._energy_slope
+
+    def _define_hardcoded_energy_ranges(self):
+        """
+        Define the hardcoded energy ranges for the various primaries.
+        """
+
+        energy_ranges = defaultdict(dict)
+        energy_ranges["gamma"][20] = [3 * u.GeV, 330 * u.TeV]
+        energy_ranges["gamma"][40] = [6 * u.GeV, 660 * u.TeV]
+        energy_ranges["gamma"][60] = [12 * u.GeV, 990 * u.TeV]
+
+        for zenith_angle in energy_ranges["gamma"]:
+            energy_ranges["gamma_diffuse"][zenith_angle] = energy_ranges["gamma"][zenith_angle]
+            energy_ranges["electron"][zenith_angle] = energy_ranges["gamma"][zenith_angle]
+
+        energy_ranges["proton"][20] = [8 * u.GeV, 600 * u.TeV]
+        energy_ranges["proton"][40] = [12 * u.GeV, 800 * u.TeV]
+        energy_ranges["proton"][60] = [16 * u.GeV, 1800 * u.TeV]
+
+        energy_ranges["helium"][20] = [10 * u.GeV, 1200 * u.TeV]
+        energy_ranges["helium"][40] = [20 * u.GeV, 2400 * u.TeV]
+        energy_ranges["helium"][60] = [40 * u.GeV, 3600 * u.TeV]
+
+        energy_ranges["nitrogen"][20] = [40 * u.GeV, 4000 * u.TeV]
+        energy_ranges["nitrogen"][40] = [80 * u.GeV, 8000 * u.TeV]
+        energy_ranges["nitrogen"][60] = [160 * u.GeV, 12000 * u.TeV]
+
+        energy_ranges["silicon"][20] = [50 * u.GeV, 5000 * u.TeV]
+        energy_ranges["silicon"][40] = [100 * u.GeV, 10000 * u.TeV]
+        energy_ranges["silicon"][60] = [200 * u.GeV, 15000 * u.TeV]
+
+        energy_ranges["iron"][20] = [60 * u.GeV, 6000 * u.TeV]
+        energy_ranges["iron"][40] = [120 * u.GeV, 12000 * u.TeV]
+        energy_ranges["iron"][60] = [240 * u.GeV, 18000 * u.TeV]
+
+        return energy_ranges
+
+    def _define_hardcoded_number_of_showers(self):
+        """
+        Define the hardcoded number of showers for the various primaries.
+        """
+
+        number_of_showers = defaultdict(dict)
+        number_of_showers["gamma"][20] = 5000
+        number_of_showers["gamma"][40] = 5000
+        number_of_showers["gamma"][60] = 2500
+
+        for zenith_angle in number_of_showers["gamma"]:
+            number_of_showers["gamma_diffuse"][zenith_angle] = number_of_showers["gamma"][
+                zenith_angle
+            ]
+            number_of_showers["electron"][zenith_angle] = number_of_showers["gamma"][zenith_angle]
+
+        number_of_showers["proton"][20] = 15000
+        number_of_showers["proton"][40] = 15000
+        number_of_showers["proton"][60] = 7500
```

Comment:
Why less at 60? The increased energy threshold?

Reply:
Yes, less events can be simulated in 24 hours (the limit on the grid), because they are higher energy events.